doc/maintenance/development_versions.rst

```diff
@@ -0,0 +1,46 @@
+.. _development_versions:
+
+How to Maintain Package Versions
+================================
+
+Sometimes, typically when proposing a change (e.g., modifying the Backend
+↔ Frontend protocol, or adding new methods into the
+``python3-copr-common`` package that other packages depend on), we need to
+specify version constraints between our components to avoid
+misinstallations.  Below are a few documented scenarios.
+
+Changing the Frontend ↔ Backend Protocol
+----------------------------------------
+
+There are two relevant values: ``MIN_FE_BE_API`` (backend side) and
+``Copr-FE-BE-API-Version`` (frontend side).
+
+Whenever the backend detects that the Frontend counterpart is older than
+expected, it delays processing.  Please bump the value anytime you think
+it's warranted—there's no risk in doing so, but forgetting to update it is
```

Comment:
Nit: I'd go a comma and a new sentence after "warranted". The dash is slightly confusing.